src/scss/config/color/_ccs-defaults.scss

```diff
@@ -0,0 +1,11 @@
+/// ## CCS Defaults
+/// ------------
+/// The Cascading Color System has a few settings
+/// that we can most easily override with CSS variables
+/// @colors ccs-colors
+/// @group config-color
+
+/* stylelint-disable-next-line dollar-variable-pattern */
+$ccs-neutral--theme: var(--ccs-h--prime);
```

Comment:
@jgerigmeyer @mirisuzanne the linter yells at me even after I added this line so I am not sure what to do instead?

Reply:
Fixed in https://github.com/oddbird/oddleventy/commit/fc17e225d2eb008cc0b5208e327db137ea0de80f